sentry-spring/src/main/java/io/sentry/spring/tracing/SentrySpan.java

```diff
@@ -19,21 +19,21 @@
    *
    * @return description
    */
-  @AliasFor("value")
   String description() default "";
 
   /**
    * Span operation.
    *
    * @return operation.
    */
+  @AliasFor("value")
   String operation() default "";
 
   /**
    * Span description.
    *
    * @return description.
    */
-  @AliasFor("description")
+  @AliasFor("operation")
   String value() default "";
```

Comment:
not sure if this is right, `AliasFor`= operation, javadocs is `description` and name of the param is `value`.
I'd double check this class and its methods, aliases and javadocs

Reply:
Fixed.